2-mandatory/5-writing-tests.js

```diff
@@ -0,0 +1,76 @@
+/*
+  For this exercise, the function has been written for you and you need to write
+  the tests.
+
+  This function takes a marking coursework score and converts it into a string
+  representing a grade (from A to E). It then returns the grade.
+*/
+
+function convertScoreToGrade(score) {
+  let grade = null;
+
+  if (score >= 80) {
+    grade = "A";
+  } else if (score >= 70) {
+    grade = "B";
+  } else if (score >= 60) {
+    grade = "C";
+  } else if (score >= 50) {
+    grade = "D";
+  } else {
+    grade = "E";
+  }
+
+  return grade;
+}
+
+/* ======= TESTS - FOR THIS EXERCISE YOU SHOULD MODIFY THEM! =====
+- To run the tests for this exercise, run `npm test -- --testPathPattern 5-writing-tests.js`
+- To run all exercises/tests in the mandatory folder, run `npm test`
+- (Reminder: You must have run `npm install` one time before this will work!)
+*/
+
+/*
+  The first test has been written for you. You need to fix the test so that it
+  passes.
+*/
+test("a score of 83 is grade A", () => {
+  expect(convertScoreToGrade(83), "Z");
+});
+
+/*
+  The rest of the tests have comments describing what to test and you need to
+  write a matching test
+*/
+
+/*
+  Write a test that checks a score of 71 is grade B
+*/
```

Comment:
I wonder if the first of these could be provided with slightly more structure to ease into writing everything themselves:
```suggestion
test.skip("a score of 71 is grade B", () => {
});
```

Reply:
Thanks. I've accepted this, plus added instructions about removing the `.skip`.